src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("19.0.2.0")]
-[assembly: AssemblyFileVersion("19.0.2.0")]
+[assembly: AssemblyVersion("19.1.0.0")]
```

Comment:
`AssemblyVersion` remains the same for minor version updates, only the `AssemblyFileVersion` changes